tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceUITest.java

```diff
@@ -249,9 +249,6 @@ public void testReplaceAllInSelection() {
 		fTextViewer.setSelection(new TextSelection(4 + System.lineSeparator().length(), 8 + System.lineSeparator().length()));
 		initializeFindReplaceUIForTextViewer();
 
-		dialog.unselect(SearchOptions.GLOBAL);
-		dialog.unselect(SearchOptions.WHOLE_WORD);
-
```

Comment:
These changes seem unrelated. Can they be reverted?

Reply:
Both tests run with these changes added back in.
Semantically they don't change test (as our current UIs just implicitly change the global-option when there is a multiline selection on opening the UI)

Reply:
Reverted in https://github.com/eclipse-platform/eclipse.platform.ui/commit/2f9136a25069244769073ee77ad5be15b5dadeee

Reply:
Yes, these changes can be reverted.

1) they don't impact our unit tests
2) our UIs implicitly enable these options anyway when opened with a multiline selection. These lines of code are helpful in the case that we introduce a third find/replace ui which does not behave like that.